django/db/backends/base/features.py

```diff
@@ -157,8 +157,8 @@ class BaseDatabaseFeatures(object):
     # Support for the DISTINCT ON clause
     can_distinct_on_fields = False
 
-    # Can the backend use an Avg aggregate on DurationField?
-    can_avg_on_durationfield = True
+    # Can the backend use an aggregate on DurationField?
+    can_agg_on_durationfield = True
```

Comment:
Agreed. I'll remove the flag, and if we find a 3rd party backend having issues we can add it back.